src/commands/CreateBanListCommand.ts

```diff
@@ -38,12 +38,11 @@ export async function execCreateListCommand(roomId: string, event: any, mjolnir:
         "redact": 50,
         "state_default": 50,
         "users": {
-            // populated in a moment
+            [await mjolnir.client.getUserId()]: 100,
```

Comment:
I think that this should be `await mjolnir.client.getUserId()` and `event["sender"]`, without the surrounding brackets.